src/components/PastTours/SearchBar.tsx

```diff
@@ -1,10 +1,37 @@
-import React from 'react'
+import React, { useState } from 'react'
+import './PastTours.css'
+
+
+interface pastTour {
+  id: number;
+  date: string;
+  location: string;
+  creator_id: number;
+  complete: boolean;
+}
+
+interface searchProps {
+  filterTours: (input: string) => pastTour[];
+}
+
+export const SearchBar: React.FC<searchProps> = ({filterTours}) => {
+  const [input, setInput] = useState<string>('')
+
+const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+  setInput(event.target.value)
+  filterTours(event.target.value)
+}
 
-export default function SearchBar() {
   return (
-    <div>
-      
-    </div>
+    <form className='search-input'>
+      <input
+        type='text'
+        name='input'
+        placeholder='Search By Location'
+        value={input}
+        onChange={(event) => handleChange(event)}
+      />
```

Comment:
Made a note of this in the new issue I created to make this work without matching uppercase, but I've done some user testing in the past on mobile search bars and some people get confused when there's not an actual "enter" button next to the input.